loadtesting/proxy/common/base.py

```diff
@@ -89,7 +89,6 @@ def setup(self) -> None:
         # create new account for each simulating user
         self.account = self.web3_client.create_account()
```

Comment:
I disagree with this changes because You broke all behavior for proxy tests.
I think You could override this method in tracer TaskSets and remove self.account from list.
Or make composition from methods like:
1. prepare_account - in proxy append account to a shared list
2. get_recipient - in proxy get random user, in tracer - create new

and override only small methods